tutorial/doppleganger.js

```diff
@@ -0,0 +1,529 @@
+"use strict";
+//======================================
+// Version 1.1
+// Addaption to "Local" Entities (since Overlays get deprecated for "Model" type.)
+// by Alezia Kurdis on February 2020
+//======================================
+// Version 1.0
+//  doppleganger.js
+//
+//  Created by Timothy Dedischew on 04/21/2017.
+//  Copyright 2017 High Fidelity, Inc.
+//  Copyright 2022 Overte e.V.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//
+/* global module */
+// @module doppleganger
+//
+// This module contains the `Doppleganger` class implementation for creating an inspectable replica of
+// an Avatar (as a model directly in front of and facing them).  Joint positions and rotations are copied
+// over in an update thread, so that the model automatically mirrors the Avatar's joint movements.
+// An Avatar can then for example walk around "themselves" and examine from the back, etc.
+//
+// This should be helpful for inspecting your own look and debugging avatars, etc.
+//
+// The doppleganger is created as an overlay so that others do not see it -- and this also allows for the
```

Comment:
```suggestion
// The doppleganger is created as an overlay so that others do not see it -- this also allows for the
```

Reply:
This comment was probably there coded into this...
Actually I would replace "overlay" by "local entity"
I will change this soon.